### DISCUSSION TITLE
Help with the script

### DISCUSSION THREAD

**Discussion by Marinzok** [WithSecureLabs/chainsaw#119] [Q&A]
Hi everyone!
I need help with building the script. I'm new to the script world.
Last time I got a ready script and I ran the Chainsaw with it. Can someone explain to me from the beggining what I need to do in order to get this work? I'm using Win10 X64.

Thank You.

**Comment by alexkornitzer** [comment]
Hi @Marinzok,

I am not sure I fully understand your question, would you be able to elaborate?

What do you mean by script here:
- Running chainsaw via a script?
- Buliding chainsaw?
- Writing a custom mapping file?
- Running Chainsaw from the command line?

All the best,

Alex
